### ISSUE TITLE
Build failure during pip install scandir==1.6 on macOS High Sierra

### ISSUE THREAD

**Issue by underyx** [benhoyt/scandir#90]
Output of `pip install scandir==1.6`: https://paste.ee/p/uggRw

`pip install scandir==1.5` works properly.

I get the same error on both Python 2.7.14 and 3.6.2.

```
$ clang -v
Apple LLVM version 9.0.0 (clang-900.0.37)
Target: x86_64-apple-darwin17.0.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
```

**Comment by benhoyt** [comment]
I don't understand why this is happening. It still looks to me like there's something wrong with your build setup.

It's complaining about stdint.h including itself:

```
In file included from /usr/local/Cellar/python/2.7.14/Frameworks/Python.framework/Versions/2.7/include/python2.7/Python.h:42:
  In file included from /usr/include/stdlib.h:65:
  In file included from /usr/include/sys/wait.h:110:
  In file included from /usr/include/sys/resource.h:72:
  In file included from /usr/local/include/stdint.h:59:
  In file included from /usr/local/include/stdint.h:59:
  In file included from /usr/local/include/stdint.h:59:
```

The actual scandir diff between 1.5 and 1.6 is here: https://github.com/benhoyt/scandir/compare/v1.5...v1.6 Anything that looks to you like it would cause problems? The C code has only very minor changes, no `#include` differences or anything.

It works on my macOS Sierra box under both Python 2.7.10 and Python 3.6. The only difference I can spot is is the clang InstalledDir:

```
$ clang -v
Apple LLVM version 9.0.0 (clang-900.0.37)
Target: x86_64-apple-darwin16.7.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

**Comment by underyx** [comment]
At the time of 1.6's release I migrated to a new macbook actually, which
might've somehow messed up my setup. I'll look into it more tomorrow.

**Comment by underyx** [comment]
Has to be a High Sierra problem, I have a coworker who pointed out that `pycurl==7.43.0` also has this issue, and I can reproduce that on my laptop too.

**Comment by benhoyt** [comment]
Got it, that's annoying. Yeah, I'm still on Sierra (macOS 10.12.6). Please post any update you find here!

**Comment by underyx** [comment]
I've finally managed to figure this one out! It's not related to `scandir` at all. Somehow High Sierra migration messed up my include directories, left crappy header files in `/usr/local/include`, and then this directory gained precedence over all the others when `clang` ran.

I ran this to get the list of headers Homebrew doesn't recognize and move them away, not sure yet if this will break anything else:

```
mkdir /tmp/includes
brew doctor 2>&1 | ag "/usr/local/include" | awk '{$1=$1;print}' | xargs -I _ mv _ /tmp/includes
```

Either way, issue closed. Thanks for having a look at it!
